tests/integration/Owncloud/OcisPhpSdk/OcisPhpSdkTestCase.php

```diff
@@ -112,6 +112,12 @@ protected function getAccessToken(string $username, string $password): string
         return $accessTokenResponse['access_token'];
     }
 
+    protected function getOcis(string $username, string $password): Ocis
```

Comment:
i think inside this file there are some place where we can use this function?
```
        $token = $this->getAccessToken($username, $password);
        return new Ocis($this->ocisUrl, $token, ['verify' => false]);
```

Reply:
Good point, thank you. I would suggest to do that in a different PR. I've created an issue, so that we don't for get it #104